.github/workflows/apps-link-checker.yml

```diff
@@ -10,12 +10,14 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        app: [ff-site, ffdweb-site]
+        app: [ff-site, ffdweb-site, filecoin-site]
         include:
           - app: ff-site
             base_url: https://fil.org
           - app: ffdweb-site
             base_url: https://ffdweb.org
+          - app: filecoin-site
+            base_url: https://filecoin.io
```

Comment:
@barbaraperic - This would now run on the current filecoin.io site, right?

Reply:
Perhaps, for the moment, we want it to run on `https://filecoin-site.vercel.app/`, right? We can add a todo to the [Redesign & Migration](https://www.notion.so/filecoin/filecoin-io-V3-EPIC-Redesign-Migration-20d7631f282580608251fa76745f4301?v=1f57631f282580479623000c9d2a4bce&source=copy_link) ticket.

Reply:
I'll add the follow up note 👍